Animixplay/src/main/kotlin/com/hexated/Animixplay.kt

```diff
@@ -130,9 +131,29 @@ class Animixplay : MainAPI() {
         }
     }
 
-    override suspend fun quickSearch(query: String) = search(query)
 
-    override suspend fun search(query: String): List<SearchResponse>? {
+    override suspend fun search(query: String):List<SearchResponse>?{
+        val searchData = app.post(
+            url = "https://v1.ij7p9towl8uj4qafsopjtrjk.workers.dev",
+            referer = mainUrl,
+            data = mapOf(
+                "q2" to query,
+                "origin" to "1",
+                "root" to "animixplay.to",
+                "d" to "gogoanime.tel")
+        )
+        return Jsoup.parse(JSONObject(searchData.text).get("result").toString()).select("div").map{elem->
```

Comment:
change it to smth like this : 
```kt
 return app.post(
            url = "https://v1.ij7p9towl8uj4qafsopjtrjk.workers.dev",
            referer = mainUrl,
            data = mapOf(
                "q2" to query,
                "origin" to "1",
                "root" to "animixplay.to",
                "d" to "gogoanime.tel")
        ).parsedSafe<FullSearch>()?.result?.let {
            Jsoup.parse(it).select("div").map { elem ->
```

add data class on bottom
```kt
private data class FullSearch(
@JsonProperty("result") val result: String? = null,
)
```

Reply:
> change it to smth like this :
> 
> ```kotlin
>  return app.post(
>             url = "https://v1.ij7p9towl8uj4qafsopjtrjk.workers.dev",
>             referer = mainUrl,
>             data = mapOf(
>                 "q2" to query,
>                 "origin" to "1",
>                 "root" to "animixplay.to",
>                 "d" to "gogoanime.tel")
>         ).parsedSafe<FullSearch>()?.result?.let {
>             Jsoup.parse(it).select("div").map { elem ->
> ```
> 
> add data class on bottom
> 
> ```kotlin
> private data class FullSearch(
> @JsonProperty("result") val result: String? = null,
> )
> ```

You can't do that because then the entire provider breaks with many errors and then it can't ressolve anime